File: Lee.EMS/8.1.0/vue/src/store/entities/dictionary.ts

/*-------------------------------------------------------
// Copyright (C) 2020 Lic
//
// 功能描述:公共字典  实体对象
//
//
// 创建标识: Lee -- 2023-07-29 22:36:02
//
//------------------------------------------------------*/
 
import Entity from './entity'
// 公共字典 
export default class Dictionary extends Entity<number>{
    name:string;
    sort:number;
    code:string;
    mem:string;
}

File: Lee.EMS/8.1.0/aspnet-core/src/Lee.EMS.Core/BAS/Dictionary.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Abp.Domain.Entities.Auditing;

namespace Lee.EMS.BAS
{
    [Table("Bas_Dictionary")]
    public class Dictionary:AuditedEntity<long>
    {
        [Column("ParentId")]
        public long Parentid { get; set; }

        [Column("Name")]
        [Required]
        [StringLength(100)]
        public string Name { get; set; }

        [Column("Sort")]
        public int Sort { get; set; }

        [Column("Code")]
        [StringLength(36)]
        public string Code { get; set; }

        [Column("Mem")]
        [StringLength(200)]
        public string Mem { get; set; }
    }
}
